docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -57,9 +72,9 @@ quarkus.http.insecure-requests=disabled
 quarkus.http.tls-configuration-name=https
 ----
 
-=== Configure a client to use https://
+=== Configuring HTTPS for a client
 
-As an example to illustrate client configuration, we will use a gRPC client:
+The following example configures a gRPC client named "hello" to use HTTPS with a truststore from the default TLS configuration:
```

Comment:
I'm inclined to having **the** gRPC client named "hello". 
The point is that if your application has a bunch of named gRPC clients, by doing this you configure one of them.
We could also keep "a", but then it should rather be
> a gRPC client (named "hello" in this example)

Reply:
Using "the" would work only if the user has a bunch of clients and one of them is reliably called "hello" - which I don't think is the case.

For comparison - "I fixed a difficult bug" (implying there are many difficult bugs and I fixed one of them), as opposed to "I fixed the difficult bug" (implying I had several bugs assigned, but only one was difficult, and I fixed it).

Reply:
> Using "the" would work only if the user has a bunch of clients and one of them is reliably called "hello" - which I don't think is the case.

What makes you think that's not the case? If there is a named client, it probably means that the user has multiple clients (if there was only one client, the user likely wouldn't have given the client a name).

Reply:
gRPC clients always have a name, even if you have a single one.

Reply:
Ah, ok, I thought there is a notion of an unnamed client in gRPC but there isn't.
Anyway my point still stands, because one name always identifies exactly one client.

Reply:
@jmartisk , grammar-wise, your own suggestion "The following example configures a gRPC client (named "hello" in this example)" almost completely overlaps with "The following example configures a gRPC client named "hello"".

Using "...configures the gRPC client named "hello"..." would only apply in a context where it is established that one of the clients is named "hello" - which doesn't really apply here, as far as I can tell. Feel free to correct me if that's not accurate, though.

Reply:
The user can name the client whatever they want. In this example, we assume that they named one client "hello". I'd say it's quite standard to invent an example name in documentation. If you prefer, we can name it something in capital letters to make it look more like a placeholder.